projects/Mallard/src/components/Button/Button.tsx

```diff
@@ -25,10 +25,12 @@ export enum ButtonAppearance {
     apricot,
     skeletonLight,
     skeletonActive,
```

Comment:
all changes to this file are part of 'darkVersion' - so not essential for just the popup